### PR TITLE
adds a new mild trauma

### PR DESCRIPTION
**Pull request by timothyteakettle**
## About The Pull Request
adds a new **[REDACTED]** to the **[CLASSIFIED]** because its funny

tested on local by giving myself the trauma and talking

## Why It's Good For The Game
**[REDACTED]**

## Changelog
:cl:
add: a new mild trauma, **[REDACTED]**
/:cl:

### CODE REVIEW

File: code/datums/brain_damage/mild.dm [hunk 1]
```diff
@@ -264,3 +264,36 @@
...
+					suffix = potential_suffix
+					break
+
+			if(suffix_foundon)
+				word = copytext(word, 1, suffix_foundon)
+
+			word = html_decode(word)
+
+			if(prob(25))
+				word = pick(list("<b>\[REDACTED]</b>", "<b>\[CLASSIFIED]</b>"))
```
Comment (zeroisthebiggay) [inline code/datums/brain_damage/mild.dm]:
```suggestion
				word = pick(list("<b>\[REDACTED]</b>", "<b>\[CLASSIFIED]</b>", "<b>\[EXPLETIVES REDACTED]</b>", "<b>\[EXPUNGED]</b>"))
```
lol two isn't funny enough
more more more more

also are you sure the bold tags work in chat or should you be using +[REDACTED]+ for the text parsing instead for those

Comment (timothyteakettle) [reply code/datums/brain_damage/mild.dm]: the bold tags work in chat, i've given the trauma to myself and tested it

Comment (timothyteakettle) [reply code/datums/brain_damage/mild.dm]: EXPLETIVES REDACTED seems weird but i'll add EXPUNGED

**Comment by buffyuwu** [comment]
oh and i just did an scp funny meme pr myself very cool <3

**Comment by silicons** [comment]
rp server character trauma

**Pull request merged**